NEMO/admin.py

```diff
@@ -210,7 +210,7 @@ class ToolAdmin(admin.ModelAdmin):
 	actions = [duplicate_tool_configuration]
 	form = ToolAdminForm
 	fieldsets = (
-		(None, {'fields': ('name', 'parent_tool', '_category', 'qualified_users', '_post_usage_questions'),}),
+		(None, {'fields': ('name', 'parent_tool', '_description', '_image', '_category', 'qualified_users', '_post_usage_questions'),}),
```

Comment:
```suggestion
		(None, {'fields': ('name', 'parent_tool', '_category', 'qualified_users', '_post_usage_questions'),}),
		('Additional Information', {'fields': ('_description', '_image'),}),
```
I think it would be nicer to have this in a separate section